lib/hooks/api/useContributors.ts

```diff
@@ -48,7 +48,7 @@ const useContributors = (intialLimit = 10, repoIds: number[] = [], range = 30) =
     query.delete("topic");
   }
 
-  query.set("range", `${range}`);
+  query.set("range", `${range ?? 30}`);
```

Comment:
nit: set the default value above

`const { pageId, selectedFilter, range = 30 } = router.query;`